addons/delivery/models/stock_picking.py

```diff
@@ -89,6 +89,8 @@ def _compute_shipping_weight(self):
     package_ids = fields.Many2many('stock.quant.package', compute='_compute_packages', string='Packages')
     weight_bulk = fields.Float('Bulk Weight', compute='_compute_bulk_weight')
     shipping_weight = fields.Float("Weight for Shipping", compute='_compute_shipping_weight')
+    carrier_tracking_url = fields.Char(string='Tracking Url', copy=False)
```

Comment:
Make it a computed field that depends on carrier_ref and carrier_id . It should be cleaner than having a static "maybe computed" field.